packages/components/src/select/__tests__/Select.test.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import * as React from 'react';
+import { shallow } from 'react-native-testing-library';
+
+import Select from '../Select';
+
+const optionsData = {
+  oneWay: {
+    icon: 'flight-direct',
+    label: 'One way',
+  },
+  return: {
+    icon: 'flight-return',
+    label: 'Return',
+  },
+};
+
+it('renders', () => {
+  expect(
+    shallow(
+      <Select
+        optionsData={optionsData}
+        selectedType={Object.keys(optionsData)[0]}
+        onSelect={jest.fn()}
+        onClosePress={jest.fn()}
+      />,
+    ),
+  ).toMatchSnapshot();
+});
```

Comment:
Nice, but can you add also test for onSelect (callback)? Test if onSelect will be call after the click and if is called with the proper value.
Als0 simple test fo `onClosePress()`

Reply:
Ok, both tests added.